src/main/java/com/code_intelligence/jazzer/mutation/mutator/libfuzzer/LibFuzzerMutators.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.code_intelligence.jazzer.mutation.mutator.libfuzzer;
+
+import com.code_intelligence.jazzer.mutation.api.ChainedMutatorFactory;
+import com.code_intelligence.jazzer.mutation.api.MutatorFactory;
+
+public final class LibFuzzerMutators {
+  private LibFuzzerMutators() {}
+
+  public static MutatorFactory newFactory() {
+    return new ChainedMutatorFactory(new FuzzedDataProviderMutatorFactory());
```

Comment:
Do we need to wrap the `FuzzedDataProviderMutatorFactory` into a `ChainedMutatorFactory`? With a single child, it should not be any  different from the original factory.

Reply:
It's not required. Consistency to the other `Mutators` classes seemed nice, though. Removed the `ChainedMutatorFactory`.